### PR TITLE
[PROD-1627] Add test random picture page

### PR DESCRIPTION
**Pull request by kezeng2024**
# context 
https://nimble-technology.atlassian.net/browse/PROD-1627
add new pages to let users to generate pictures by 3 fine tuned model.
three pages as below
1 test-page
2 test-twitter-summary-page
3 test-cartoon-pokeman-page

# test

Screenshot 2024-04-23 at 1 27 51 PM

test-page
Screenshot 2024-04-23 at 6 08 30 PM

test-twitter-summary-page
Screenshot 2024-04-24 at 10 39 24 AM

test-cartoon-pokeman-page
Screenshot 2024-04-24 at 10 38 51 AM

### CODE REVIEW

File: src/pages/test-cartoon-pokeman-page.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+        var textElement = document.getElementById("randomCategory");
+
+        textElement.textContent = images[randomIndex][1];
+    }
+    
+    return (
+        <Box sx={{width: '100%', marginTop: '270px'}}>
+            <Box sx={{width: '50%', marginTop: '55px', marginLeft: '300px', fontSize: '28px'}}>
+                Please click Random button to try images by cartoon pokermon model generation!
```
Comment (ejipeng) [inline src/pages/test-cartoon-pokeman-page.js:58]: Change to generate as discussed.

File: src/pages/test-cartoon-pokeman-page.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            </Box>
+            <Box sx={{width: '20%', marginTop: '10px', marginLeft: '100px'}}>
+
+            <Button sx={backButtonStyle} onClick={showRandomPicture}>
+                      Random
```
Comment (ejipeng) [inline src/pages/test-cartoon-pokeman-page.js:63]: ditto

File: src/pages/test-twitter-summary-page.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            <Button sx={backButtonStyle} onClick={showRandomSummary}>
+                      Random
+            </Button>
+            </Box>
+
+            <Box sx={{width: '50%', marginTop: '55px', marginLeft: '300px'}}>
+                <Box id="randomImage" component="img" sx={{ width: 500, height: 700, }} />
+            </Box>
+
+            <Box id="randomCategory" component="a" sx={{width: '60%', marginTop: '55px', marginLeft: '200px', fontSize: '18px'}}>
```
Comment (ejipeng) [inline src/pages/test-twitter-summary-page.js:64]: Add summary title to the box and change the summary text layout according to device type/browser size.

File: src/components/header-component.js [hunk 4]
```diff
@@ -30,6 +30,8 @@ const Header = () => {
             navigate("/tokenomics")
         } else if (value === SECTIONS.NEWS) {
             navigate('/media/')
+        } else if (value === SECTIONS.TEST) {
```
Comment (ejipeng) [inline src/components/header-component.js:33]: Remove this in this PR and add to the ecosystem page as discussed

File: src/pages/test-cartoon-pokeman-page.js (hunk 1 above)
Comment (kezeng2024) [reply src/pages/test-cartoon-pokeman-page.js:58]: would update

File: src/pages/test-cartoon-pokeman-page.js (hunk 2 above)
Comment (kezeng2024) [reply src/pages/test-cartoon-pokeman-page.js:63]: would update

**Comment by LazyBoyJgn99** [comment]
## Update

### test-nft
image

### test-cartoon-pokeman
image

### test-twitter-summary
image

**Pull request merged**